includes/wp-admin/table/trait-actor-list-table.php

```diff
@@ -48,4 +48,23 @@ public static function get_webfinger( $actor ) {
 
 		return Webfinger::guess( $actor );
 	}
+
+	/**
+	 * Get the action URL for a follower.
+	 *
+	 * @param string $action   The action.
+	 * @param string $follower The follower ID.
+	 * @return string The action URL.
+	 */
+	private function get_action_url( $action, $follower ) {
+		return \wp_nonce_url(
+			\add_query_arg(
+				array(
+					'action'   => $action,
+					'follower' => $follower,
+				)
+			),
+			$action . '-follower_' . $follower
```

Comment:
Any mention of `follower` here should probably be `actor` to be more broadly applicable? However, changing this would also mean that `process_action()` needs to have all its parameters updated.